_pages/en/strategy-build-vs-buy.md

```diff
@@ -132,31 +149,48 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 
 ## Measuring the Strategy's success
 
-This Strategy's success will be measured by comparing the following metrics against the key elements of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604).
-
-- Adoption/Building
-  - Increase adoption of open source software components
-  - Increase contributions to third party OSS projects
-  - Decrease number of forked versions of OSS unaligned with upstream or without contributions of changes
-  - Increase number of internal projects released as OSS
-  - Increase external contributions to ESDC OSS projects
-- Buy
-  - Decrease lock-in situations (need to define how measure)
-  - Decrease use of non-interoperable and exclusive:
-    - file formats
-    - APIs
-    - data-exports
-  - Increase number of workloads migrated to cloud environments (Cloud First Strategy)
+This Strategy's success will be measured by comparing the following metrics against the key elements of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and on [Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604).
+
+Specific to Mandatory Procedures for Enterprise Architecture:
+
+- Upcoming
+
+Specific to Mandatory Procedures for Application Programming Interface:
+
+- Upcoming
+
+Specific to acquisition of new or enhancements to IT Solutions and IT Products:
+
+- Adopting
+  - Increased ratio of open source software solutions and products used in production compared to purchased or internally developed ones.
+  - Increased contributions to third-party OSS projects instead of internally maintained changes.
+  - Decreased ratio of forked versions of third-party OSS unaligned with upstream or without contributions of changes.
+  - Increased number of contributions to ESDC OSS projects from the community.
+- Buying
+  - Decreased ratio of IT Solutions and IT Products in used in production by ESDC for which the department can not make software changes due to:
+    - Contract obligations with vendors
+    - Intellectual property limitations
+    - Proprietary and closed formats
+  - Decreased ratio of IT Solutions and IT Products for which the department can not choose a service integrator and/or a software maintainer other than the group that provides the software licence.
+  - Decreased use of IT Solutions and IT Products for which there are non-interoperable and exclusive:
+    - File formats
+    - Application Programming Interfaces
+    - Data exports
+- Building
+  - Increased ratio of internally built projects released as OSS
 - Applicable to all scenarios
-  - Increased speed and flexibility for changes
+  - Increased ratio of decoupled IT Solutions and IT Products compared to baseline
```

Comment:
the challenge here will be to define what is decoupled. Maybe adding it in the definition section (I'm looking at how to define it too so the TSDM should point to this strategy). 

Maybe:

A solution or product is decoupled when:
- it can run independently in production from other product/solutions (at least 24h)
- it's only integration with other product/solutions is via an API contract

Reply:
Will need to add nuance to the concept of IT Solutions Definition itself :thinking:

Reply:
I don't have a good answer for the metrics, hard to cautiously identify them. Was asked to add more metrics, and to have less :man_shrugging: